### ISSUE TITLE
BUG: Flow Launcher won't show up

### ISSUE THREAD

**Issue by samve000** [Flow-Launcher/Flow.Launcher#2693]
### Checks

- [X] I have checked that this issue has not already been reported.

- [X] I am using the latest version of Flow Launcher.

### Problem Description

When I press Alt+Space the search bar won't show up.
There's still a flow launcher tray icon but I can't right-click it to shut it down or go to settings.
I can also not open Flow.Launcher.exe.

When I shut it down via task manager and reopen it, it starts working again, but it is not a permanent fix

### Flow Launcher Version

1.18.0

### Windows Build Number

22631.3527

**Comment by onesounds** [comment]
Need more information. Right-clicking on the tray doesn't bring up the menu, right?
In this case, the process is stuck, so please share the replay case and logs as best you can so we can verify.

**Comment by j-a-s-o-n-g** [comment]
There are many times where the latest Flow Launcher becomes unresponsive for me as well, usually after adding or modifying modifying a web search configuration in the settings UI. It isn't 100% of the time, but I do have to "end task" on it and re-open for it to behave again. Task manager doesn't indicate it isn't responsive it just doesn't work by the trigger or even right clicking the tray icon.

**Comment by samve000** [comment]
> Need more information. Right-clicking on the tray doesn't bring up the menu, right? In this case, the process is stuck, so please share the replay case and logs as best you can so we can verify.

That's correct, right clicking the tray icon does not bring up the context menu.

Here's the log from yesterday:
[2024-05-07.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/15245097/2024-05-07.txt)

**Comment by deefrawley** [comment]
Is this everytime you run Flow?

**Comment by samve000** [comment]
Not every time, it does this at random. 
Sometimes it works, sometimes it doesn't. 
Or sometimes it works in the beginning but then stops working after a while.

**Comment by jjw24** [comment]
> > Need more information. Right-clicking on the tray doesn't bring up the menu, right? In this case, the process is stuck, so please share the replay case and logs as best you can so we can verify.
> 
> That's correct, right clicking the tray icon does not bring up the context menu.
> 
> Here's the log from yesterday: [2024-05-07.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/15245097/2024-05-07.txt)

I don't see anything unusual in the logs. We will have to be a bit patient with this one until we can have consistent reproducible steps to troubleshoot this.

**Comment by deefrawley** [comment]
Hi @samve000 - does this problem persist with the most recent version of Flow?

**Comment by Jack251970** [comment]
@samve000 

Could you please check if this can help? I think it is related to window freezing.

https://ci.appveyor.com/project/JohnTheGr8/flow-launcher/builds/52753088
